src/sample/models/jepa.py

```diff
@@ -131,7 +110,7 @@ def forward(
             Encoded latents with shape [batch_size, n_patches, out_dim]
         """
         # Patchify input images
-        x = self.patch_embed(images)
+        x = self.patchfier(images)
```

Comment:
`Encoder`の入力として与えられるのってimageだけとは限らない気がする
```suggestion
        x = self.patchfier(input_data)
```

Reply:
修正しました。 https://github.com/MLShukai/pamiq-vrchat/pull/50/commits/52b6e92d3d4dc506e5d2fdc013ea1451d7abd201